templates/gin/files/server.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	_ "github.com/go-sql-driver/mysql"
+	psh "github.com/platformsh/config-reader-go/v2"
+	sqldsn "github.com/platformsh/config-reader-go/v2/sqldsn"
+	"net/http"
+)
+
+var db = make(map[string]string)
+
+func setupRouter(config *psh.RuntimeConfig) *gin.Engine {
```

Comment:
This trick of passing the config to setupRouter will work for here, but it won't work for larger applications, I think.  It may or may not work for the language-examples site.  That's... probably OK for now so not a merge blocker, but we may want to refactor this in the future.